share/rocm/cmake/ROCMSphinxDoc.cmake

```diff
@@ -79,6 +89,7 @@ function(rocm_add_sphinx_doc SRC_DIR)
         ${PROJECT_NAME}-sphinx-${BUILDER}
         COMMAND
             "${SPHINX_EXECUTABLE}"
+            ${CONFIG_DIR_ARG}
```

Comment:
Should this be quoted?

Reply:
While it's going to need extra testing, for the time being, it worked for me on Linux when the source tree was accessed through a symlink with a space in it.